src/Game/Scenes/LoginScene.cs

```diff
@@ -510,8 +510,18 @@ private void NetClient_PacketReceived(object sender, Packet e)
             switch (e.ID)
             {
                 case 0xA8: // ServerListReceived
+                    //Save account information if current login step is Verifying account.
+                    //Otherwise, we are backing from character selection and we dont want to resave.
+                    if(CurrentLoginStep == LoginSteps.VerifyingAccount)
+                    {
+                        AccountManager.SaveAccount(Settings.GlobalSettings.IP, Settings.GlobalSettings.Username, Settings.GlobalSettings.Password, Settings.GlobalSettings.SaveAccount);
+                        if (!Settings.GlobalSettings.SaveAccount)
```

Comment:
This is questionable as it appears to move away from the current behavior.  I am just not sure if the current functionality to keep these fields populated if the user does not have the "Save Account" checkbox enabled and then go back to the login screen is desired.  It seems like a potential current bug, but this may be original functionality.  Could be easily removed.